client/stylesheets/_responsive.scss

```diff
@@ -92,6 +107,37 @@
     // .searchResultsCardLi {
     // 	width: 90%;
     // }
+
+    // .profileCardImg {
+    //     height: 115px !important;
+    // }
+
+    .managerOfferView {
+        margin-top: 4px !important;
+    }
+
+    .profileCardInfo > li:first-child {
+        margin-bottom: 6px !important;
+    }
+}
+
+@media (max-width: 876px) {
+
+    .managerOfferView {
+        margin-top: 20px !important;
+    }
+
+    .profileCardInfo > li:first-child {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileCardInfo` should be written in lowercase with hyphens